infra/gcp/ensure-conformance-storage.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to create a new "conformance" bucket in GCS.
+#
+# Each conformance bucket exists in its own GCP project, and is writable by a
+# dedicated googlegroup.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
+    echo "usage: $0 [repo...]" > /dev/stderr
+    echo "example:" > /dev/stderr
+    echo "  $0 # do all staging repos" > /dev/stderr
+    echo "  $0 coredns # just do one" > /dev/stderr
+    echo > /dev/stderr
+}
+
+# NB: Please keep this sorted.
+CONFORMANCE_PROJECTS=(
+    capi-openstack
+)
+if [ $# = 0 ]; then
+    # default to all staging projects
+    set -- "${CONFORMANCE_PROJECTS[@]}"
+fi
+
+for REPO; do
+    color 3 "Configuring staging: ${REPO}"
+
+    # The GCP project name.
+    PROJECT="k8s-conformance-${REPO}"
```

Comment:
minor point: there's a limited character count available in some contexts, and "conformance" eats a lot of that.  I suggest "cnfm" or "c9e" or something

Reply:
this is reflecting the current naming scheme for the google.com owned buckets, it was fairly arbitrary at the time.

Reply:
I would use "cnfm" but only for the Project I guess? Shouldn't be an issue on the Google group?

Reply:
I forget the exact limits, but ISTR the list name had to be less than 30 characters, too - "capi-foobar" came about because "cluster-api-provider-foobar" overlfowed.

I value consistency - if we train our brain to read "cnfm" as "conform", let's do it everywhere.

"cnfm" is clunkly.  "cfrm" maybe better?  "cnfrm" is as long as "infra".  "conform" is as long as "staging" - that seems OK and parallel.

Reply:
Good point. So let's use conform